### PR TITLE
feat: make the poller poll multiple pricefeeds

### PR DESCRIPTION
**Pull request by girazoki**
This PR adds support for specifying mutliple pricefeeds in the TOML configuration file, and as long as they share the same ABI the poller will interact with all of them

### CODE REVIEW

File: contract.py [hunk 1]
```diff
@@ -12,11 +12,13 @@ def wrb(w3, config_file):
...
+  contracts = []
     # ABI generated using
     # https://remix.ethereum.org
   with open("pricefeed_abi.json") as json_file:
     contract_abi = json.load(json_file)
-    contract = w3.eth.contract(contract_address, abi=contract_abi)
-    return contract
+    for address in contract_addresses:
+      contracts.append(w3.eth.contract(address, abi=contract_abi))
+    return contracts
```
Comment (aesedepece) [inline contract.py:23]: The former pythonista inside of me is saying:
```suggestion
    return [ w3.eth.contract(address, abi=contract_abi) for address in contract_addresses ]
```

File: tomls/rinkeby.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [contract]
 # The contracts the poller needs to interact with
 wrb = "0x996Be500EBF09537EDde024f70fFdFA55089E939"
-pricefeed = "0xE046EC0088c9330Ecb4A90D4F2c29cA0Bc31706d"
+pricefeeds = ["0xE50372951AE5B4EF099076E9091CF1D2AF00d81E", "0xeFd4bbe486bBD5248D842337B88151B2Dd5534aA"]
```
Comment (aesedepece) [inline tomls/rinkeby.toml:4]: Which contracts are these? Can't tell from Etherscan :sweat_smile:

Comment (girazoki) [reply tomls/rinkeby.toml:4]: PriceFeeds examples. Now you should be able to tell what they are as I already verified them. I just used them to check whtat everything works smooth with a public testnet

File: contract.py [hunk 3]
```diff
@@ -12,11 +12,11 @@ def wrb(w3, config_file):
 
 # Return the pricefeed contract, given a Ethereum address from "config.toml"
 def pricefeed(w3, config_file):
-  contract_address = config_file["contract"]["pricefeed"]
+  contract_addresses = config_file["contract"]["pricefeeds"]
+  contracts = []
```
Comment (aesedepece) [inline contract.py:16]:
```suggestion

```

**Pull request merged**